python/paddle/fluid/dygraph/dygraph_to_static/basic_api_transformer.py

```diff
@@ -56,7 +54,7 @@ def visit_Expr(self, node):
             if isinstance(child_node, gast.Call):
                 # TODO(liym27):
                 #  Considers that a dygraph api which modifies the input or has a output.
-                if is_dygraph_api(child_node):
+                if utils.is_dygraph_api(child_node):
```

Comment:
Should we add `is_paddle_api` here?

Reply:
No, here it only needs to judge whether it a dynamic graph api. 
However, there are still risks in this code, as discussed offline, I will optimize it next PR.

Reply:
No, here it only needs to judge whether it a dynamic graph api.
However, there are still risks in this code, as discussed offline, I will optimize it next PR as stated in the PR comments.